quint/testFixture/_1028importCycleC.json

```diff
@@ -1 +1 @@
-{"stage":"parsing","warnings":[],"errors":[{"explanation":"Cyclic imports: 'mocked_path/testFixture/_1028importCycleC.qnt' imports 'mocked_path/testFixture/_1026importCycleA.qnt' imports 'mocked_path/testFixture/_1027importCycleB.qnt' imports 'mocked_path/testFixture/_1028importCycleC.qnt'","locs":[{"source":"mocked_path/testFixture/_1027importCycleB.qnt","start":{"line":1,"col":2,"index":20},"end":{"line":1,"col":45,"index":63}}]}]}
\ No newline at end of file
+{"stage":"parsing","warnings":[],"errors":[{"explanation":"extraneous input 'from' expecting {'}', 'const', 'var', 'assume', 'type', 'val', 'def', 'pure', 'action', 'run', 'temporal', 'import', DOCCOMMENT}","locs":[{"source":"mocked_path/testFixture/_1028importCycleC.qnt","start":{"line":1,"col":20,"index":38},"end":{"line":1,"col":23,"index":41}}]}]}
```

Comment:
@konnov seems like your `update-fixtures` command picked up an old version of quint without `from` and `export`

Reply:
yeah, I have run `npm run generate` again, which uses the `npm` version